app.js/App.js

```diff
@@ -0,0 +1,61 @@
+
+
+
+'use strict'
+
+
+
+function Employee(EmployeeID,FullName,Department,Level,ImageURL,Salary){
```

Comment:
Great work Lith,
one note: 
* use small letters for params and properties names